dl/pytorch/0.1.11/tests/nn.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
Thanks for adding support for tests. This will really help in ensuring the quality of future images.